config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: [:show, :update]
```

Comment:
Based on the comment above about RESTful design: Instead of using the `:update` for dishes, consider this routing to allow you to create a path with both ingredients and dishes and route it to a `:create` action
```
resources :dishes, only: :show do
     resources :ingredients, only: :create, controller: "dish_ingredients"
end
```
So, then the path for your form would be `POST to "/dishes/:dish_id/ingredients"` and it would land in a DishIngredientsController in the create action